src/js_native_api.h

```diff
@@ -21,6 +21,10 @@
 #endif
 #endif
 
+#if defined(NAPI_EXPERIMENTAL) && !defined(NODE_API_EXPERIMENTAL_NO_WARNING)
```

Comment:
We can use `NODE_WANT_INTERNALS` to suppress the warning in Node.js builds, instead of changing the CI scripts.

```suggestion
#if defined(NAPI_EXPERIMENTAL) && !defined(NODE_API_EXPERIMENTAL_NO_WARNING) && !defined(NODE_WANT_INTERNALS)
```

Reply:
Thanks, just pushed that